srml/executive/src/lib.rs

```diff
@@ -652,4 +696,134 @@ mod tests {
 		execute_with_lock(WithdrawReasons::all());
 		execute_with_lock(WithdrawReasons::except(WithdrawReason::TransactionPayment));
 	}
+
+	#[test]
+	fn block_hooks_weight_is_stored() {
+		new_test_ext(0).execute_with(|| {
+
+			Executive::initialize_block(&Header::new_from_number(1));
+			// NOTE: might need updates over time if system and balance introduce new weights. For
+			// now only accounts for the custom module.
+			assert_eq!(<system::Module<Runtime>>::all_extrinsics_weight(), 150 + 25);
+		})
+	}
+}
+
+#[cfg(test)]
```

Comment:
This is some separate test that I actually ended up integrating into the main `mod tests`.  But at some point it had a compile error that I still want to investigate and lear why it was failing. I will remove this later. 

See the code at the bottom:  when calling `Executive::initialize_block(&Header::new_from_number(1));` it would complain that `Executive` does not have the function because the `UncheckedExtrnisic` type is not `Checkable`, which in my opinion perfectly is.

```
error[E0277]: the trait bound `sr_primitives::generic::unchecked_extrinsic::UncheckedExtrinsic<srml_indices::address::Address<substrate_primitives::sr25519::Public, u32>, tests_weights::Call, substrate_primitives::sr25519::Signature, ()>: sr_primitives::traits::Checkable<srml_system::ChainContext<tests_weights::Runtime>>` is not satisfied
   --> srml/executive/src/lib.rs:769:4
    |
769 |             <UncheckedExtrinsic as Checkable<ChainContext<Runtime>>>::check(a, &Default::default());
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `sr_primitives::traits::Checkable<srml_system::ChainContext<tests_weights::Runtime>>` is not implemented for `sr_primitives::generic::unchecked_extrinsic::UncheckedExtrinsic<srml_indices::address::Address<substrate_primitives::sr25519::Public, u32>, tests_weights::Call, substrate_primitives::sr25519::Signature, ()>`
    |
    = help: the following implementations were found:
              <sr_primitives::generic::unchecked_extrinsic::UncheckedExtrinsic<Address, Call, Signature, Extra> as sr_primitives::traits::Checkable<Lookup>>
```